openid-4-verifiable-presentations-1_0.md

```diff
@@ -2339,6 +2339,8 @@ Setting `limit_disclosure` property defined in [@!DIF.PresentationExchange] to `
 
 A non-normative example of the Authorization Response would look the same as in the examples of other Credential formats in this Annex.
 
+The Key Binding JWT `nonce` claim MUST be the value of `nonce` from the authorization request, and the `aud` claim MUST be the Client Identifier of the Verifier.
```

Comment:
```suggestion
The following are the requirements for the values of `nonce` and `aud` claims in the Key Binding JWT:

- the `nonce` claim MUST be the value of `nonce` from the authorization request;
- the `aud` claim MUST be the value of `client_id` from the authorization request

When `client_id` parameter is absent in an unsigned OpenID4VP request sent over the DC API, the `aud` claim MUST be the value of origin received from the platform.
```

two points I am trying to clarify:
- two normative statements should not be in the same sentence
- behavior when client id is not present, e.g., unsigned browser API request, needs to be defined

Reply:
"Client Identifier" in the case of an unsigned browser request is defined here:

https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#appendix-A.2

> The client_id parameter MUST be omitted in unsigned requests defined in [Appendix A.3.1](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#unsigned_request). The Wallet determines the effective Client Identifier from the origin as asserted by the Web Platform and/or app platform. The effective Client Identifier is composed of a synthetic Client Identifier Scheme of web-origin and the origin itself. For example, an origin of https://verifier.example.com would result in an effective Client Identifier of web-origin:https://verifier.example.com.

I'd prefer to rely on this text (which is what my current text does) instead of adding MUSTs that duplicate existing clauses.

Reply:
```suggestion
The following are the requirements for the values of `nonce` and `aud` claims in the Key Binding JWT:

- the `nonce` claim MUST be the value of `nonce` from the Authorization Request;
- the `aud` claim MUST be the value of `client_id` from the Authorization Request, unless it is an unsigned request over the DC API, where the `aud` claim MUST be the value of origin received from the platform;

```

Reply:
@jogu then please accept Paul's suggestion. even if not a must statement we need a clarification here.

Reply:
```suggestion
The following requirements apply to the `nonce` and `aud` claims in the Key Binding JWT:

- the `nonce` claim MUST be the value of `nonce` from the Authorization Request;
- the `aud` claim MUST be the value of the Client Identifier; 

Note that for an unsigned Authorization Request over the DC API the `client_id` parameter is not used and the effective Client Identifier is derived from the origin received from the platform as described in (#dc_api_request).
```